volttron/platform/agent/base_historian.py

```diff
@@ -416,6 +419,10 @@ def __init__(self,
             STATUS_KEY_CACHE_FULL: False
         }
         self._all_platforms = bool(all_platforms)
+        self._time_tolerance = float(time_tolerance) if time_tolerance else None
```

Comment:
what should happen if invalid input...if float time_tolerance raises an error...I think the agent should die.

Reply:
If there is  invalid values for any parameter base historian init will fail and hence agent will not start. 
Updated init to also check type of time_tolerance_topics parameter. Updated configure method to also do similar check. If bad value is sent for config update using config store then, config update logs exception and fails but agent continues to run with the previous configurations